### ISSUE TITLE
Design Intent - Accessibility Feedback - Accredited Representation Management

### ISSUE THREAD

**Issue by briandeconinck** [department-of-veterans-affairs/va.gov-team#66388]
## VFS actions
- [ ] Assign this ticket to the team member(s) responsible for addressing feedback provided by Platform.
- [ ] Comment on this ticket:
  - [ ] If the Platform reviewer has any **Thoughts/Questions** that require responses.
  - [x] When **Must** feedback has been incorporated. As appropriate, link to any other GitHub issues or PRs related to this feedback.
  - [x] When **Should/Consider** feedback has been incorporated, or if any feedback will not be addressed. As appropriate, link to any other GitHub issues or PRs related to this feedback.
- [x] Close the ticket when all feedback has been addressed.

## Thoughts/questions

- Lots of great CAIA feedback already documented! Rather than repeat everything @coforma-jamie has already flagged I'll just highlight a few items.

## Feedback
Practice areas will document their feedback on the VFS-provided artifacts following the [Must, Should, and Consider Framework](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/must-should-and-consider-framework-for-feedback). Platform reviewers may also provide additional notes that don’t comment on the artifacts themselves but are important for implementation (eg. engineering/coding notes).

- **Must:** PDF accessibility! I think I muddled this a little in the meeting, so just to hopefully make it a little clearer:
   - As @artsymartha68 noted, the blank PDF version of the form is generated using tools maintained by a different team and are not your responsibility.
   - But the finished product from your workflow --- writing to a PDF to provide a completed document that can be submitted via mail --- is where you'll need to test for accessibility. Basically, you want to make sure that the document generated by your product is intelligible and navigable for screen reader users, so that they can review their document after completing your workflow. I believe Jeana Clark on the Veteran Facing Forms team may have some useful experience around this, or may be able to point you in the right direction. 
- **Must:** Maps are a huge accessibility headache, and there are a few things I'll encourage you to be thinking about with the results page:
   - Design for desktop, mobile, and 200%, 300%, and 400% zoom. High zoom doesn't give you a lot of real estate, so this is a big challenge!
   - Be very deliberate about keyboard interactions, and in particular the tab order for a keyboard user navigating through the search results. What do you have to tab through to get to the next thing (viewing a result, getting to the pagination links, etc)?
- **Should:** Whenever possible, try to avoid conditionally revealing information or new form fields. We like "one thing per page" because it often helps reduce cognitive load, and because conditionally-revealing things can be tricky to code for assistive technology. Sometimes it makes sense to do it anyway, but if there's a way to break it on to its own page that's usually going to be my preference.
- **Should:** Jamie flagged some specific button/link items in your lo-fi wireframes, but I'll give general guidance here for moving forward:
   1. With a few exceptions, we want to maintain material honesty. If something is coded as a `button`, it should look like a button. If something is coded as an `a`, it should look like a link.
   2. If an element is navigational (changes the URL in the address bar of the browser), it should be a link. If an element is an action (fires some JavaScript to change content on the page, or saves user information by some mechanism), it should be a button.

**Comment by oddball-lindsay** [comment]
Full write up from @coforma-jamie lives here: https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/teams/CAIA/accessibility/drafts/accredited-rep-management.md

**Comment by oddball-lindsay** [comment]
Michael is working to include this feedback in his prototype work: https://app.zenhub.com/workspaces/accredited-representation-management-team-64d0dc51d3e8f4788ac6ef96/issues/gh/department-of-veterans-affairs/va.gov-team/66347

**Comment by oddball-lindsay** [comment]
Updated designs, ready for Midpoint review: https://www.sketch.com/s/dfb0333d-65db-41f9-807e-6b25d6c5396e/p/3C666192-27DA-40CE-B314-BA779A8C8BD4/canvas#About
